### PR TITLE
[4.0] webservices xml

### PR DESCRIPTION
**Pull request by brianteeman**
Fix copy paste error in the initial commit on the creationDate

code review only

### CODE REVIEW

File: plugins/webservices/content/content.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <extension type="plugin" group="webservices" method="upgrade">
 	<name>plg_webservices_content</name>
 	<author>Joomla! Project</author>
-	<creationDate>August 2017</creationDate>
```
Comment (wilsonge) [inline plugins/webservices/content/content.xml:5]: Technically this does date back to 2017 in the original gsoc repo (https://github.com/joomla-projects/gsoc19_webservices/commit/7967ebb99749dbccbf2936d68cf39d12e9a17abd#diff-5602e4d1807527ff3a4e471013f9da778a935ff16a46d565def37d30e69808b1 ). Not really sure whether to use merge date here or the “real” date there

Comment (brianteeman) [reply plugins/webservices/content/content.xml:5]: If the creation date is correct then the copyright date is wrong ;)

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 387ded9904949728468602ea5d2129abd84b027c<br /><br />Simple text change. Looks good.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31544">issues.joomla.org/tracker/joomla-cms/31544</a>.</sub>

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 387ded9904949728468602ea5d2129abd84b027c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31544">issues.joomla.org/tracker/joomla-cms/31544</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31544">issues.joomla.org/tracker/joomla-cms/31544</a>.</sub>

File: plugins/webservices/content/content.xml (hunk 1 above)
Comment (HLeithner) [reply plugins/webservices/content/content.xml:5]: @brianteeman do you like to change the copyright date instead of the creation date?

Comment (brianteeman) [reply plugins/webservices/content/content.xml:5]: @HLeithner that would also mean changing the copyright dates in the php file

Comment (HLeithner) [reply plugins/webservices/content/content.xml:5]: @nibra can you please have a look what happens to the copyright dates for the webservices in the copyright PR?

Comment (nibra) [reply plugins/webservices/content/content.xml:5]: The year numbers in the copyright statement were retrieved from git history. When those dates do not fit (e.g., files where copied, then changed, or the like) feel free to set them to an more appropriate value. Technically, it doesn't matter. The year number has merely documentation purpose. Some projects started to omit them completely.

The sole intention to use the start date only was and is to get rid of the yearly copyright year bump.

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
thx

**Pull request merged**